spartan/metrics/values/prod.yaml

```diff
@@ -40,21 +40,21 @@ prometheus:
   server:
     resources:
       requests:
-        memory: 26Gi
+        memory: 40Gi
         cpu: "3.5"
     nodeSelector:
       node-type: infra
-      pool: spot
-    tolerations:
-      - key: "cloud.google.com/gke-spot"
-        operator: "Equal"
-        value: "true"
-        effect: "NoSchedule"
+    #   pool: spot
+    # tolerations:
+    #   - key: "cloud.google.com/gke-spot"
+    #     operator: "Equal"
+    #     value: "true"
+    #     effect: "NoSchedule"
```

Comment:
The otel-collector expects there to be a prom instance collocated on the same machine. I'm assuming this will be a single node in the infra pool?

https://github.com/AztecProtocol/aztec-packages/blob/master/spartan/metrics/values/prod.yaml#L24

Reply:
LE: yep, got the TF part of the PR and there's only one infra node. This will work fine

Reply:
Oh that's interesting. They were on different machine before and it didn't seem to cause a problem